src/tests/ftest/SConscript

```diff
@@ -0,0 +1,23 @@
+"""Build ftest"""
+import os
+
+def scons():
+    """Execute build"""
+    Import('env', 'prereqs')
+
+    ftest_install_dir = '$PREFIX/lib/daos/TESTING/ftest'
+
+    env.Install(ftest_install_dir, Glob('*.*'))
+
+    dirs = ['container', 'control', 'daos_test', 'data', 'io', 'network', \
+            'object', 'pool', 'rebuild', 'security', 'server', 'soak', \
+            'unittest']
+
+    for dir in dirs:
```

Comment:
dir is used by python.  I think if pylint was working it would warn here.   Please name it something else

Reply:
oh, changed to sub_dir to avoid future pylint warning.